README.md

```diff
@@ -15,6 +15,12 @@ but the `PersistentVolume` objects will have a [NodeAffinity][node affinity] con
 
 ![architecture with Hostpath](architecture.hostpath.drawio.svg "Architecture with Hostpath provisioning")
 
+As a third option, if the ZFS host is part of the cluster, you can let the provisioner choose
+whether [NFS][nfs] or [HostPath][hostpath] is used with the `Auto` mode. If the scheduler
+decides to place a Pod onto the ZFS host, *and* the requested access mode in the Persistent Volume
+Claim is `ReadWriteOnce` (the volume can only be accessed by pods running on the same node)
+[HostPath][hostpath] will automatically be used, otherwise [NFS][nfs] will be used.
+
```

Comment:
```suggestion
As a third option, if the ZFS host is part of the cluster, you can let the provisioner choose
whether [NFS][nfs] or [HostPath][hostpath] is used with the `Auto` mode. If the scheduler
decides to place a Pod onto the ZFS host, *and* the requested access mode in the Persistent Volume
Claim contains `ReadWriteOnce` (the volume can only be accessed by pods running on the same node)
[HostPath][hostpath] will automatically be used, otherwise [NFS][nfs] will be used.

```